src/install/ai-monitoring/agent-lang/go-config.mdx

```diff
@@ -0,0 +1,43 @@
+---
+componentType: default
+headingText: Configure the Go agent
+freshnessValidatedDate: never
+---
+
+To collect AI data, you need to ensure that certain configurations are enabled when you instrument your AI-powered app. We also recommend updating other variables to improve data retention.
+
+* We recommend applying these configurations in the New Relic config file. 
```

Comment:
The Go agent does not use a config file. It is configured using functions.
We recommend applying these configurations in your `newrelic.NewApplication` function call by passing the `newrelic.ConfigAIMonitoringEnabled`, `newrelic.ConfigAIMonitoringStreamingEnabled`, and `newrelic.ConfigAIMonitoringRecordContentEnabled` functions as parameters to `NewApplication`. If you included the `newrelic.ConfigFromEnvironment` option function, you may instead configure these options using environment variables.